### DISCUSSION TITLE
How to batch rename

### DISCUSSION THREAD

**Discussion by QYQSDTC** [sxyazi/yazi#530] [1. Q&A] [answered]
After selecting multiple files in Visual mode and pressing r, nothing happens. How to batch rename like the feature video shows.

**Comment by sxyazi** [comment, marked as answer]
What opener are you using for? if you're using the default opener, check whether your `$EDITOR` is set, see https://yazi-rs.github.io/docs/faq#why-am-i-getting-a-permission-denied-when-editing-files-in-linuxmacos

**Reply by QYQSDTC** [reply]
I set the EDITOR to nvim, now after selecting multiple files, pressing r will enter this screen
![CleanShot-8UDQTB1O@2x](https://github.com/sxyazi/yazi/assets/15608306/361730db-eeff-4246-b974-9f429439865a)

But what's the next step?

**Reply by sxyazi** [reply]
Just edit them as what you want then `:wq`
